examples/examples_test.go

```diff
@@ -67,12 +67,8 @@ func getBaseOptions(t *testing.T) integration.ProgramTestOptions {
 			"aws-native:region": envRegion,
 			"prefix":            prefix,
 		},
-		// some flakiness in some resource creation
-		// @see https://github.com/pulumi/pulumi-aws-native/issues/1714
-		RetryFailedSteps:     true,
-		ExpectRefreshChanges: true,
 		SkipRefresh:          true,
-		Quick:                true,
+		ExpectRefreshChanges: true,
```

Comment:
We've fixed most of these issues, but there is still an issue with refresh changes with IAM roles due to https://github.com/pulumi/pulumi-cdk/issues/293